src/tests/ftest/get_remote_files.sh

```diff
@@ -161,7 +161,7 @@ scp_files() {
                 fi
             fi
         else
-            echo "  Failed to archive ${file} to ${2}"
+            echo "  Failed to archive ${file} to ${2} (error code: $?"
```

Comment:
Are you missing a closing `)` in the message?  `${PIPESTATUS[0]}` is preferred over `$?`:
```suggestion
            echo "  Failed to archive ${file} to ${2} (error code: ${PIPESTATUS[0]})"
```